File: Selenium/Activities/Actiivty10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains

# Set up the driver
driver = webdriver.Chrome()
actions = ActionChains(driver)

# Open the drag and drop page
driver.get("https://training-support.net/webelements/drag-drop")

# Print the title of the page
print("Page title:", driver.title)

# Find the ball
ball = driver.find_element(By.ID, "ball")

# Find Dropzone 1 and Dropzone 2
dropzone1 = driver.find_element(By.ID, "dropzone1")
dropzone2 = driver.find_element(By.ID, "dropzone2")

# Drag and drop the ball into Dropzone 1
actions.drag_and_drop(ball, dropzone1).perform()
print("Ball entered Dropzone 1:", dropzone1.text)

# Drag and drop the ball into Dropzone 2
actions.drag_and_drop(ball, dropzone2).perform()
print("Ball entered Dropzone 2:", dropzone2.text)

# Close the browser
driver.quit()

File: Selenium/Activities/Actiivty10.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10 {
    public static void main(String[] args) {
        // Set up the driver
        WebDriver driver = new ChromeDriver();
        Actions actions = new Actions(driver);

        // Open the drag and drop page
        driver.get("https://training-support.net/webelements/drag-drop");

        // Print the title of the page
        System.out.println("Page title: " + driver.getTitle());

        // Find the ball
        WebElement ball = driver.findElement(By.id("ball"));

        // Find Dropzone 1 and Dropzone 2
        WebElement dropzone1 = driver.findElement(By.id("dropzone1"));
        WebElement dropzone2 = driver.findElement(By.id